### PR TITLE
Organisation dashboard content and design fixes

### LINKED ISSUE #354: Organisation dashboard: Content and design fixes
## Organisation dashboard

Image
Image

- [x] Org name in breadcrumb needs correct capitalisation
- [x] Dataset list should be alphabetically ordered
- [x] "Not submitted" tag should be grey
- [ ] "Live" tag shows alongside text showing issues exist / either the tag or the text is wrong
- [x] When there's only a single issue the text should be "There is 1 issue in this dataset"

### PR DESCRIPTION
**Pull request by DilwoarH**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Fixes the following content issues:
- Org name in breadcrumb needs correct capitalisation
- Dataset list should be alphabetically ordered
- "Not submitted" tag should be grey
- "Live" tag shows alongside text showing issues exist / either the tag or the text is wrong
- When there's only a single issue the text should be "There is 1 issue in this dataset"

## Related Tickets & Documents

- Closes #354 

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

### CODE REVIEW

**Review by GeorgeGoodall — DISMISSED** [review]
~~lg2m~~
never mind, still a WIP

File: test/unit/filters/pluralise.test.js
Comment (rosado) [inline test/unit/filters/pluralise.test.js:1]: I don't think there's much point in this test - this is basically testing the pluralize library, which already has tests like this: https://github.com/plurals/pluralize/blob/master/test.js

**Review by rosado — CHANGES_REQUESTED** [review]

File: test/unit/filters/pluralise.test.js
Comment (DilwoarH) [reply test/unit/filters/pluralise.test.js:1]: I've added a missing test.

While I do agree mostly, I think we should keep it in. While it's mostly a wrapper, I think we should have tests to cover the basic logic of the filter (where we pass no count and it returns singular).

Comment (rosado) [reply test/unit/filters/pluralise.test.js:1]: There's no "logic" there, the parameter is passed explicitly or via default parameter.

Also, after having a look at it again:

- `pluralise` is not invoked with one parameter anywhere outside of test code
- it doesn't seem intuitive to have function named `pluralise` which by default returns a singular form 😉 

Given the two points above, I don't think we should have that wrapper at all. It's confusing to use and adds a bunch of tests for a code path that we never even traverse.

**Review by rosado — CHANGES_REQUESTED** [review]

File: test/unit/filters/pluralise.test.js
Comment (DilwoarH) [reply test/unit/filters/pluralise.test.js:1]: Agreed, good points - I've removed the wrapper and just call it directly now

**Review by rosado — APPROVED** [review]

**Pull request merged**